src/resources/elements/primeDesignSystem/pform-item/pform-item.ts

```diff
@@ -0,0 +1,13 @@
+import {customElement} from "aurelia-framework";
+import "./pform-item.scss";
+import {bindable} from "aurelia-typed-observable-plugin";
+import {ValidationState} from "../types";
+
+@customElement("pform-item")
+export class PFormItem {
+  @bindable public validationState?: ValidationState;
+  @bindable label = "";
+  @bindable labelExtra = "";
```

Comment:
hey @hiaux0  I just red this comment but I can't find the `public` keyword because this file doesn't exist anymore. It was renamed to `pform-input`. Nonetheless, I removed `public` from here.